.github/workflows/ci-backend-cql.yml

```diff
@@ -63,30 +63,54 @@ jobs:
           - module: cql
             args: "-Pcassandra3-byteordered -Dtest=\"**/diskstorage/cql/*\""
             name: byteordered-diskstorage
+            java: 8
           - module: cql
             args: "-Pcassandra3-murmur -Dtest=\"**/diskstorage/cql/*\""
             name: murmur-diskstorage
+            java: 8
           - module: cql
             args: "-Pcassandra3-byteordered -Dtest=\"**/graphdb/cql/*\""
             name: byteordered-graphdb
+            java: 8
           - module: cql
             args: "-Pcassandra3-murmur -Dtest=\"**/graphdb/cql/*\""
             name: murmur-graphdb
+            java: 8
           - module: cql
             args: "-Pcassandra3-murmur -Dtest=\"**/hadoop/*\""
             name: murmur-hadoop
+            java: 8
           - module: cql
             args: "-Pcassandra3-byteordered -Dtest=\"**/core/cql/*\""
             name: byteordered-core
+            java: 8
           - module: cql
             args: "-Pcassandra3-murmur -Dtest=\"**/core/cql/*\""
             name: murmur-core
+            java: 8
           - module: cql
             args: "-Pcassandra3-murmur-ssl -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
             name: murmur-ssl
+            java: 8
           - module: cql
             args: "-Pcassandra3-murmur-client-auth -Dtest=\"**/diskstorage/cql/CQLStoreTest.java\""
             name: murmur-client-auth
+            java: 8
+          - module: cql
+            install-args: "-Pjava-11"
+            args: "-Pcassandra3-murmur -Pjava-11 -Dtest=\"**/diskstorage/cql/*\""
+            name: murmur-diskstorage-java-11
+            java: 11
+          - module: cql
+            install-args: "-Pjava-11"
+            args: "-Pcassandra3-murmur -Pjava-11 -Dtest=\"**/graphdb/cql/*\""
+            name: murmur-graphdb-java-11
+            java: 11
+          - module: cql
+            install-args: "-Pjava-11"
+            args: "-Pcassandra3-murmur -Pjava-11 -Dtest=\"**/hadoop/*\""
+            name: murmur-hadoop-java-11
+            java: 11
```

Comment:
Why for Java 11 we test only `cassandra3-murmur` and not `cassandra3-byteordered` or `cassandra3-murmur-ssl` or `cassandra3-murmur-client-auth`?

Reply:
I don't want copy all tests.

Reply:
The same for hbase and the others.

Reply:
I believe you don't need to copy / paste all the tests. I think you can just add java version to matrix as a separate parameter instead of included parameter and it should run all versions of java (i.e. 8 and 11) with all tests.
Here is I found an example: https://ncorti.com/blog/howto-github-actions-build-matrix#example-detekt
In that example he is using the next strategy to execute all combinations of Java (8, 11, 14) and OSes:
```
jobs:
  gradle:
    strategy:
      fail-fast: false
      matrix:
        os: [ubuntu-latest, macos-latest, windows-latest]
        jdk: [8, 11, 14]
```

So, I don't think you need to add `java: 8` and `java: 11` to all the tests. Instead, we can just move them to matrix parameters and run all combinations.
If so, we will be able to guarantee that all our tests can be executed with Java 8 and Java 11.

Also, I would suggest to include `[tp-tests]` to this PR so that we could check if everything really works with Java 11.

Reply:
I did it before as matrix. Not everything is compatible with java 11 such as our full tests. I had issue with the huge test groups because can just rerun a single check.

If we would like to do it just as a martix, i think we refactor the maven poms a bit more. 
My idea was to have just some test for java 11, as mark it as experimental.

Reply:
I don't see an advantange to run everything against java 8 and java 11.